R/linear-mixed-model-deployment.R

```diff
@@ -30,9 +29,32 @@
 #' F leads to removal of rows containing NULLs.
 #' @param debug Provides the user extended output to the console, in order
 #' to monitor the calculations throughout. Use T or F.
+#' @param cores Number of cores you'd like to use.  Defaults to 2.
```

Comment:
Same comment as in Lasso docs... probably should make example consistent with default (unless you want to show that you can indicate one core in the example, knowing the default is 2 cores).